challengeutils/challenge.py

```diff
@@ -1,8 +1,173 @@
-import logging
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__)
+"""Synapse Challenge Services"""
+import json
+from typing import Union
 
+from synapseclient import Project, Synapse, Team
+try:
+    from synapseclient.core.utils import id_of
+except ModuleNotFoundError:
+    # For synapseclient < v2.0
+    from synapseclient.utils import id_of
 
-class Challenge(object):
-    def __init__(self, syn):
+from .synapseservices.challenge import Challenge
+
+
+class ChallengeApi:
+    """Challenge services
+    https://docs.synapse.org/rest/index.html#org.sagebionetworks.repo.web.controller.ChallengeController
+
+    Args:
+        id: The ID of this Challenge object
+        project: synapseclient.Project or its id
+        team: synapseclient.Team or its id
+
```

Comment:
We should probably keep the whitespace consistent, e.g. some docstrings have a blank line before the closing triple-quotes while others don't.

Reply:
Agreed, thanks.  It is actually pep8 standards to have a blank line before the closing.

Reply:
Oh, I thought it was blank line after the closing?

Reply:
It appears we are both wrong: https://google.github.io/styleguide/pyguide.html